### PR TITLE
Add a choice to import archived Trello cards or not

### PR DESCRIPTION
**Pull request by jerryskye**
Hello! Thank you for creating this project :blush:
I noticed that the Trello JSON contains archived cards. I've added support for choosing whether we want to import the archived items or not. Let me know what you think :wink:
Stay awesome! :wave:

### CODE REVIEW

File: src/importers/trelloJson/index.ts [hunk 1]
```diff
@@ -21,4 +25,10 @@ const questions = [
     name: 'trelloFilePath',
     message: 'Select your exported JSON file of Trello cards',
   },
+  {
+    type: 'confirm',
+    name: 'importArchived',
+    message: 'Would you like to import the archived cards as well?',
```
Comment (rlargman) [inline src/importers/trelloJson/index.ts:31]:
```suggestion
    message: 'Would you like to import the archived cards as well (not recommended)?',
```

File: src/importers/trelloJson/index.ts [hunk 2]
```diff
@@ -21,4 +25,10 @@ const questions = [
...
+    default: true,
```
Comment (rlargman) [inline src/importers/trelloJson/index.ts:32]:
```suggestion
    default: false,
```

**Review by rlargman — CHANGES_REQUESTED** [review]
Good catch here! One small suggestion: I think it makes sense to default to _not_ import the archived issues rather than the reverse, as that is the general recommendation.

**Comment by jerryskye** [comment]
Thank you for your review! I didn't want to change the default behaviour, but this makes more sense :blush: I think I'm going to change the question to `'Would you like to discard the archived cards?'`

**Review by rlargman — APPROVED** [review]
Excellent, thanks again!

**Pull request merged**